osfoffline/views/start_screen.py

```diff
@@ -1,12 +1,22 @@
+import furl
+import aiohttp
```

Comment:
Best I can say is that it's a moderately absurd solution to a very absurd situation in our active codebase. I agree that in a perfect world, we could both go back to not caring. For now, at least we'll have company in the insanity. :)